### PR TITLE
Feature : CustomerRequestDTO added

### PR DESCRIPTION
**Pull request by sagarxt**
#38

### CODE REVIEW

**Review by smruti-banty — COMMENTED** [review]
What about Response. 
- In the Response do you share customer password ?

File: backend/src/main/java/com/moshop/backend/controller/CartController.java
Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/controller/CartController.java:1]: Remove space from line 31

File: backend/src/main/java/com/moshop/backend/controller/CustomerController.java
Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/controller/CustomerController.java:1]: Use interface for Injection. don't use Impl class

**Review by smruti-banty — COMMENTED** [review]
change according to comment

File: backend/src/main/java/com/moshop/backend/controller/CustomerController.java
Comment (sagarxt) [reply backend/src/main/java/com/moshop/backend/controller/CustomerController.java:1]: changes done

**Review by Abinashkjcjanscinanconanc — APPROVED** [review]
LGTM

**Pull request merged**